infrastructure/identity-aware-proxy/main.py

```diff
@@ -0,0 +1,7 @@
+import sys
+from make_iap_request import *
+
+if __name__ == "__main__":
+	# sys.argv[1] = URL
+	# sys.argv[2] = IAP Client Id
+	print make_iap_request(sys.argv[1],sys.argv[2])
```

Comment:
You should do some basic validation on your arguments - at least ensure the arguments have been provided, and use sensible error messages if they have not.

Reply:
Done.